DependencyInjection/HBStampieExtension.php

```diff
@@ -42,15 +43,23 @@ public function load(array $configs, ContainerBuilder $container)
             throw new \InvalidArgumentException(sprintf('Invalid mailer "%s" specified', $config['mailer']));
         }
 
+        if (class_exists('Symfony\Component\DependencyInjection\ChildDefinition')) {
```

Comment:
this check is here to avoid deprecation warnings in Symfony 3.3, where the class got renamed.